http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreter.scala

```diff
@@ -1082,17 +1091,22 @@ class GKEInterpreter[F[_]](
       _ <- dbApp.app.numOfReplicas match {
         case Some(count) =>
           for {
-            attemptToStart <- kubeService
-              .patchReplicas(
-                dbApp.cluster.getClusterId,
-                KubernetesNamespace(dbApp.app.appResources.namespace),
-                KubernetesDeployment(dbApp.app.appName.value), // appNames are the same as deployments
-                count
-              )
-              .attempt
+            deployments <- kubeService.listDeployments(dbApp.cluster.getClusterId,
+                                                       KubernetesNamespace(dbApp.app.appResources.namespace)
+            )
+            attemptToStopResults <- deployments.parTraverse { deployment =>
```

Comment:
Should this be renamed to `attemptToStartResults`?

Reply:
ah yes...copy paste error :)